README.md

```diff
@@ -14,72 +15,118 @@ OSPSuite.ReportingEngine implementation in R
-    - [Installation instructions](https://github.com/Open-Systems-Pharmacology/OSPSuite-R#installation)
+You can install the development version of `{ospsuite.reportingengine}` from [GitHub](https://github.com/) with:
 
-[OPTIONAL] Install **Pandoc** (required for generation of reports in MS-Word format):
+```r
+# install.packages("remotes")
+remotes::install_github("Open-Systems-Pharmacology/OSPSuite.ReportingEngine")
+```
+### Using the package bundle &#128230;
 
-* [Pandoc Installer (Windows)](https://github.com/jgm/pandoc/releases/download/2.9.2.1/pandoc-2.9.2.1-windows-x86_64.msi)
-* [Pandoc Installer (Linux)](https://github.com/jgm/pandoc/releases/download/2.9.2.1/pandoc-2.9.2.1-linux-amd64.tar.gz)
+You can install a specific version of the `{ospsuite.reportingengine}` package by downloading and installing its zip or tar.gz bundle [here &#128230;](https://ci.appveyor.com/project/open-systems-pharmacology-ci/OSPSuite-ReportingEngine/history).
 
-[OPTIONAL] Install **rsvg-convert** (required by Pandoc for conversion of images in SVG format)
+- The package bundle for the __development__ version is available [here &#128230;](https://ci.appveyor.com/project/open-systems-pharmacology-ci/OSPSuite-ReportingEngine/branch/develop/artifacts).
+- The package bundle for the __release__ version is available [here &#128230;](https://ci.appveyor.com/project/open-systems-pharmacology-ci/OSPSuite-ReportingEngine/branch/master/artifacts).
 
-* [Installer (Windows)](https://github.com/miyako/console-rsvg-convert/releases)
-  * The installation folder must be added to the system path.
-* For Linux, **librsvg** package must be installed (package name depends on distribution, e.g. **librsvg2-bin** for Ubuntu).
+Then, to install manually, replace the `path/to/ospsuite.reportingengine.zip` by your actual local path to the `.zip` or `tar.gz` file in the code below:
 
-# Development tasks
+```r
+install.packages(path/to/ospsuite.reportingengine.zip, repos = NULL)
+```
 
-## dev_mode
+### Required packages
 
-  `devtools::dev_mode` function switches your version of R into "development mode". This is useful to avoid clobbering the existing versions of CRAN packages that you need for other tasks. Calling dev_mode() again will turn development mode off, and return you to your default library setup.
+`{ospsuite.reportingengine}` requires following packages to be installed:
 
-```R
-# This will install the package in the folder C:/Rpackages
-devtools::dev_mode(path="C:/Rpackages")
-```
+- From the [Open Systems Pharmacology](https://github.com/Open-Systems-Pharmacology) ecosystem:
 
-## Reload the package
+|Package|Version|Installation Instructions|
+|-------|-------|------------|
+|[`{ospsuite.utils}`](https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils)|$\geq$ 1.4|Download and install package bundle [here &#128230;](https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils/releases)|

```

Comment:
I assume it should be >=1.5?
e.g. validateUTF8 functions are not available in utils 1.4, correct?